lib/auth.ts

```diff
@@ -1,38 +1,60 @@
-import { expect, Page } from '@playwright/test';
```

Comment:
isn't this file duplicate of lib/pom/auth.ts?

Reply:
leftover, will delete it